### PR TITLE
Update the Security Announcements section

### PR DESCRIPTION
**Pull request by isharailanga**
- Improve language
- Fix broken links
- Security Advisory notes are checked for 2022,2021,2020,2019, 2018, 2016
   - Need to do for 2017

### CODE REVIEW

File: en/docs/security-announcements/security-advisories/2016/WSO2-2016-0170.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Apache Axis2 1.6.2 uses commons-httpclient-3.1.0 and host name verification shou
 
 
 ### IMPACT
-This allows man-in-the-middle attackers to spoof SSL servers via an arbitrary valid certificate.
+This allows Eavesdropper attackers to spoof SSL servers via an arbitrary valid certificate.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2016/WSO2-2016-0170.md:29]: Let's keep the term *man-in-the-middle* here.

File: en/docs/security-announcements/security-advisories/2017/WSO2-2017-0184.md [hunk 2]
```diff
@@ -31,10 +31,10 @@ The XXE attacks can affect any trusted system respective to the machine where th
 ### SOLUTION
 The recommended solution is to apply relevant security patch which fixes the XXE vulnerability in the WSO2 Carbon Commons component used in the listed products.
 
-Please see below for details on patching or updating the affected component.
+`See below for details on patching or updating the affected component.`
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2017/WSO2-2017-0184.md:34]: Do we need to add these inside backticks (`` ` ``)

File: en/docs/security-announcements/security-advisories/2019/WSO2-2019-0655.md [hunk 3]
```diff
@@ -27,12 +27,12 @@ Several JavaScript libraries used by the XACML entitlement user interfaces of WS
 
 
 ### IMPACT
-A malicious entity may intercept the unencrypted HTTP request used to retrieve the JavaScript content and/or alter the unencrypted HTTP response to include malicious content, in combination with other attack vectors such as man-in-the-middle attacks.
+A malicious entity may intercept the unencrypted HTTP request used to retrieve the JavaScript content and/or alter the unencrypted HTTP response to include malicious content, in combination with other attack vectors such as Eavesdropper attacks.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2019/WSO2-2019-0655.md:30]: Shall we use the `man-in-the-middle` term here.

File: en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md [hunk 4]
```diff
@@ -21,24 +21,23 @@ cvss: "7.6 (CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:U/C:H/I:H/A:L)"
 
 
 ### OVERVIEW
-A potential security vulnerability has been identified in SMS OTP Authenticatior.
+A potential security vulnerability has been identified in SMS OTP Authentication.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md:24]: I think, the word should be `authenticator`. WDYT?

File: en/docs/security-announcements/security-advisories/index.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
----
+in---
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/index.md:1]: Seems like this has been mistakenly added. WDYT?

File: en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md [hunk 6]
```diff
@@ -24,8 +24,7 @@ cvss: "8.7 (CVSS:3.0/AV:N/AC:L/PR:H/UI:N/S:C/C:H/I:N/A:H)"
 
 
 ### OVERVIEW
-It has identified that Management Console is vulnarable to a XXE attack when updating a EventPublisher.
-
+It has been identified that Management Console is vulnerable to an XXE attack when updating an EventPublisher.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md:27]: `An XXE` or `A XXE`?

Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md:27]: We can use `an` here - https://english.stackexchange.com/questions/8181/does-one-use-a-or-an-before-the-word-x-ray

File: en/docs/security-announcements/security-advisories/index.md (hunk 5 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/index.md:1]: ah yes. wil remove. thanks!

File: en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md:24]: will fix. thanks!

File: en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md (hunk 6 above)
Comment (1uffyD9) [reply en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md:27]: Noted. Thanks for the clarification.

File: en/docs/security-announcements/security-advisories/2019/WSO2-2019-0655.md (hunk 3 above)
Comment (ayomawdb) [reply en/docs/security-announcements/security-advisories/2019/WSO2-2019-0655.md:30]: This term change was part of a discussion we did around using gender terms. +1 to the new suggestion.

File: en/docs/security-announcements/security-advisories/2016/WSO2-2016-0170.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2016/WSO2-2016-0170.md:29]: Thanks all. Will use `Eavesdropper`.

File: en/docs/security-announcements/security-advisories/2022/WSO2-2021-1646.md [hunk 7]
```diff
@@ -42,7 +42,7 @@ Service Provider > Add/Edit Service Provider > Inbound Authentication Configurat
 
 Here example1 and example2 are sample URLs
 
-Moreover, the product documentations are updated with the corrected configuration samples.
```
[the remark this answers is not in the document]
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2022/WSO2-2021-1646.md:45]: The term `documentation` is an uncountable noun and therefore the plural form is `documentation` too. https://www.oxfordlearnersdictionaries.com/definition/english/documentation

File: en/docs/security-announcements/security-advisories/index.md (hunk 5 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/index.md]: Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a

File: en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md]: Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a

File: en/docs/security-announcements/security-advisories/2017/WSO2-2017-0184.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/security-announcements/security-advisories/2017/WSO2-2017-0184.md]: Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a

**Review by ayomawdb — APPROVED** [review]

**Pull request merged**